public/scripts/library/rdp/wip_rdp_abstract_sequence.js

```diff
@@ -61,6 +52,7 @@ class WipRdpAbstractSequence extends Sequence {
       // TODO: add an association for rdpEdits ?
       'rdpEdits',
       'partType',
+      'desc',
```

Comment:
We don't need this as it will come from the Sequence Factory:  https://github.com/Synbiota/GENtle2/blob/fe87a7f0b854be2bee164b49c232068c240b74d5/public/scripts/library/sequence-model/factory.js#L261